srcpkgs/upterm/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'upterm'
+pkgname=upterm
+version=0.7.6
+revision=1
+wrksrc="upterm-$version"
```

Comment:
```suggestion
```
Not necessary, `wrksrc` is `${pkgname}-${version}` by default.

Reply:
done. 👍🏼